src/components/Header/EmailButton.tsx

```diff
@@ -0,0 +1,99 @@
+import {
+  Popover,
+  PopoverContent,
+  PopoverTrigger,
+} from '@components/ui/popover';
+import { Separator } from '@components/ui/separator';
+import { Mail } from 'lucide-react';
+import Image from 'next/image';
+import Link from 'next/link';
+import { useState } from 'react';
+
+const fakeAvatar = '/header__fakeAvatar.svg';
+
+const fakeData = [
+  {
+    avatar: fakeAvatar,
+    user: 'Apple',
+    content: '您是否有任何繪畫或藝術相關的經驗？',
+    time: '2024-03-21',
+  },
+  {
+    avatar: fakeAvatar,
+    user: 'Apple',
+    content: '您是否有任何繪畫或藝術相關的經驗？',
+    time: '2024-03-21',
+  },
+  {
+    avatar: fakeAvatar,
+    user: 'Apple',
+    content: '您是否有任何繪畫或藝術相關的經驗？',
+    time: '2024-03-21',
+  },
+  {
+    avatar: fakeAvatar,
+    user: 'Apple',
+    content: '您是否有任何繪畫或藝術相關的經驗？',
+    time: '2024-03-21',
+  },
+  {
+    avatar: fakeAvatar,
+    user: 'Apple',
+    content: '您是否有任何繪畫或藝術相關的經驗？',
+    time: '2024-03-21',
+  },
+];
+
+const EmailButton = () => {
+  const [isOpen, setIsOpen] = useState(false);
+
+  return (
+    <Popover open={isOpen} onOpenChange={setIsOpen}>
+      <PopoverTrigger className="ml-[3px] mr-4 rounded-full border border-primary p-5 data-[state=open]:ml-0 data-[state=open]:mr-[13px] data-[state=open]:border-4">
+        <Mail size={24} />
+      </PopoverTrigger>
+
+      <PopoverContent className="h-0 w-0 border-none bg-transparent p-0">
+        <div className="absolute right-[-30px] top-4 w-[636px] rounded-[32px] border-4 border-black bg-[#e8e4de] pt-6 ">
```

Comment:
remove top-4, spacing would be right
```suggestion
        <div className="absolute right-[-30px] w-[636px] rounded-[32px] border-4 border-black bg-[#e8e4de] pt-6 ">
```